src/cli/commands/self-update.js

```diff
@@ -35,13 +35,6 @@ export async function run(
     timeout: 5000,
   });
 
-  // while yarn is close sourced we need an auth token to be passed
-  const githubAuth0Token = process.env.YARN_AUTH_TOKEN || process.env.KPM_AUTH_TOKEN;
-  github.authenticate({
-    type: 'oauth',
-    token: githubAuth0Token,
-  });
-
```

Comment:
This authentication was used for self-update feature only.
And it was necessary only when the repo was not public yet.